hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOmMetadataManager.java

```diff
@@ -62,16 +74,26 @@ public class TestOmMetadataManager {
 
   private OMMetadataManager omMetadataManager;
   private OzoneConfiguration ozoneConfiguration;
+  @TempDir
+  private File folder;
 
-  @Rule
-  public TemporaryFolder folder = new TemporaryFolder();
-
+  public static <E extends Throwable> void expectOmException(
```

Comment:
This is unnecessary function to me. `Assertions.assertThrows()` does the same thing.